client/errors.go

```diff
@@ -1,11 +1,24 @@
 package client
 
-import apistatus "github.com/nspcc-dev/neofs-sdk-go/client/status"
+import (
+	"errors"
+
+	apistatus "github.com/nspcc-dev/neofs-sdk-go/client/status"
+)
+
+// unwraps err using errors.Unwrap and returns the result.
+func unwrapErr(err error) error {
```

Comment:
Are there any particular reason you go with unwrapping loop instead of `errors.As`? Or is it just more convenient to unwrap errors in this code?